### PR TITLE
Goa 2715 provide ontology version and date via service

### PR DESCRIPTION
**Pull request by TonyWardell**

### CODE REVIEW

File: ontology-rest/src/main/java/uk/ac/ebi/quickgo/ontology/metadata/MetaDataConfig.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Configure the provision of metadata for the Ontology service.
+ *
+ * @author Tony Wardell
+ * Date: 07/03/2017
+ * Time: 10:55
+ * Created with IntelliJ IDEA.
+ */
+@Configuration
+@ConfigurationProperties(prefix = "metadata.ontology")
```
Comment (eddturner) [inline ontology-rest/src/main/java/uk/ac/ebi/quickgo/ontology/metadata/MetaDataConfig.java]: is this line needed? analogous line in annotation config isn't present, can it be removed if not needed (seems to conflict wtih `@ConfigurationProperties(prefix = "ontology.metadata")` in `metaDataConfigProperties`

File: ontology-rest/src/main/java/uk/ac/ebi/quickgo/ontology/metadata/MetaDataConfig.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+public class MetaDataConfig {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MetaDataConfig.class);
+    private static final Path DEFAULT_METADATA_PATH = Paths.get("ONTOLOGY_IRI.dat.gz");
+    private static final String SERVICE = "GO";
```
Comment (eddturner) [inline ontology-rest/src/main/java/uk/ac/ebi/quickgo/ontology/metadata/MetaDataConfig.java]: -> go ?

File: annotation-rest/src/main/java/uk/ac/ebi/quickgo/annotation/metadata/MetaDataConfig.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ *
+ * @author Tony Wardell
+ * Date: 07/03/2017
+ * Time: 10:55
+ * Created with IntelliJ IDEA.
+ */
+@Configuration
+public class MetaDataConfig {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MetaDataConfig.class);
+    private static final String SERVICE = "Annotation";
```
Comment (eddturner) [inline annotation-rest/src/main/java/uk/ac/ebi/quickgo/annotation/metadata/MetaDataConfig.java]: -> annotation ?

File: ontology-rest/src/main/java/uk/ac/ebi/quickgo/ontology/metadata/MetaDataConfig.java (hunk 1 above)
Comment (TonyWardell) [inline ontology-rest/src/main/java/uk/ac/ebi/quickgo/ontology/metadata/MetaDataConfig.java]: Ah yeah, left in by mistake.

**Pull request merged**